photon-core/src/main/java/org/photonvision/vision/frame/consumer/MJPGFrameConsumer.java

```diff
@@ -110,6 +111,68 @@ public class MJPGFrameConsumer implements AutoCloseable {
                 EMPTY_MAT, "Disabled", new Point(14, 45), 0, 0.6, ColorHelper.colorToScalar(Color.RED), 1);
     }
 
+    static {
```

Comment:
This content looks copy-paste between the MJPEG frame consumer and the file consumer. is it possible to condense it into a shared class? Maybe some factory that generates a "test colors bar" frame with generic text drawn on it?